src/App.jsx

```diff
@@ -14,33 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [isReversed, setIsReversed] = useState(false);
+  const [isSortedAlphabetically, setIsSortedAlphabetically] = useState(false);
+  const [isSortedByLength, setIsSortedByLength] = useState(false);
```

Comment:
try to combine it into one sortType state that will accept the string isAlphabetically or isByLength